src/dragonball/src/vm/kernel_config.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2022 Alibaba Cloud. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fs::File;
+
+/// Structure to hold guest kernel configuration information.
+pub struct KernelConfigInfo {
+    /// The descriptor to the kernel file.
+    kernel_file: File,
+    /// The descriptor to the initrd file, if there is one
+    initrd_file: Option<File>,
```

Comment:
Does this imply that normal rootfs images cannot be used? Also, what does it mean if `initrd_file == None`?

Reply:
We support normal rootfs images and initrd is for providing the possibility for any case if users need to use initrd

Reply:
rootfs image related setting will be introduced in the next PR related to block device manager